### PR TITLE
Move to kubernetes-python library

### PR DESCRIPTION
**Pull request by trnl**
- convert from pykube to official kubernetes-python
- use python:3-alpine3.7
- use requirements.txt file
- use proper logging framework

### CODE REVIEW

**Comment by dabcoder** [comment]
@hjacobs Would you have a chance to review that PR? I also think that moving to the `kubernetes-python` library makes sense there, and I'm interested in testing this cronjob with that lib. Thanks in advance.

File: cleaner.py [hunk 1]
```diff
@@ -3,106 +3,112 @@
...
 
             if seconds_since_completion > max_age:
                 return '{:.0f}s old'.format(seconds_since_completion)
 
 
 def delete_if_expired(dry_run, entity, reason):
     if reason:
-        print("Deleting {} {} ({})".format(entity.kind, entity.name, reason))
+        logger.info("Deleting %s %s (%s)", type(entity), entity.metadata.name, reason)
+        print(type(entity))
```
Comment (hjacobs) [inline cleaner.py:90]: this seems to be remaining debug output..

**Comment by dabcoder** [comment]
Thanks! @trnl, thoughts?

**Comment by hjacobs** [comment]
I would have to test the changes manually (as I never invested into unit tests :disappointed: ), I won't get around doing this soon. Generally LGTM at first glance.

**Pull request closed without merging**